src/Core/src/Handlers/View/ViewHandler.Windows.cs

```diff
@@ -1,31 +1,40 @@
 ﻿#nullable enable
 using System;
 using System.Collections.Generic;
-using System.ComponentModel;
+using System.Runtime.CompilerServices;
 using Microsoft.UI.Xaml;
-using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Input;
 using PlatformView = Microsoft.UI.Xaml.FrameworkElement;
 
 namespace Microsoft.Maui.Handlers
 {
 	public partial class ViewHandler
 	{
+		static ConditionalWeakTable<PlatformView, ViewHandler>? FocusManagerMapping;
```

Comment:
Since this's static, can't it be initialized inline?
```suggestion
		static ConditionalWeakTable<PlatformView, ViewHandler>? FocusManagerMapping = [];
```

This way we don't need to throw nothing